### PR TITLE
[6.0] Replace GMT with UTC

### PR DESCRIPTION
**Pull request by laoneo**
### Summary of Changes
Unifies the code base to use UTC instead of GMT as we offer only UTC as timezone in the options.

Code review as both timezones should be the same.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/296
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Review by HLeithner — APPROVED** [review]

**Comment by Frostholm7** [comment]
OK, test on  db9d95d7ceb350cdd2c074e4d0e092f253188ed7.
Code review, I haven't found any more of this type of change.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43912">issues.joomla.org/tracker/joomla-cms/43912</a>.</sub>

File: libraries/src/Date/Date.php [hunk 1]
```diff
@@ -100,7 +100,7 @@ public function __construct($date = 'now', $tz = null)
         // Create the base GMT and server time zone objects.
         if (empty(self::$gmt) || empty(self::$stz)) {
             // @TODO: This code block stays here only for B/C, can be removed in 5.0
-            self::$gmt = new \DateTimeZone('GMT');
+            self::$gmt = new \DateTimeZone('UTC');
```
Comment (Quy) [inline libraries/src/Date/Date.php:103]: Revert per PR #43959 ?

Comment (laoneo) [reply libraries/src/Date/Date.php:103]: Will do once the pr is merged, till then it has to stay.

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**